frontend/src/metabase/visualizations/lib/renderer_utils.js

```diff
@@ -128,9 +128,17 @@ function getParseOptions({ settings, data }) {
 }
 
 export function getDatas({ settings, series }, warn) {
+  const isHist = isHistogram(settings);
```

Comment:
Should we also filter nulls for any non-ordinal scale?

Reply:
just made that change